### DISCUSSION TITLE
`onActive` variant?

### DISCUSSION THREAD

**Discussion by loyddaniels** [btwld/mix#226] [Q&A]
What's the point of the `onPressed` variant? It just flashes when pressed. There's this, but there's no `onActive` (which would correspond to CSS's `:active` selector), in other words, when the `PressableBox` is being pressed. `onHover` doesn't work, and that seems to be only for web/desktop. How do I style `PressableBox` **while the mouse button is being held down** on it?

**Comment by AlexanderThiele** [comment]
I have the same question, why is `onTapDown` not supported? - the state whenever you press and hold the button. cc @leoafarias

**Comment by AlexanderThiele** [comment]
There was also once this issue #242 which says to use a WidgetStateController but i somehow find the active state so important to not build a custom solution around it.
